src/panels/lovelace/cards/hui-picture-elements-card.ts

```diff
@@ -1,6 +1,6 @@
 import { html, LitElement, TemplateResult } from "lit-element";
 
-import { createHuiElement } from "../common/create-hui-element";
+import { createConfiguredHuiElement } from "../../lovelace/common/validate-condition";
```

Comment:
This method should obviously not be in `common` or in `validate-condition`. It's unique to this card and the conditional card.